database/N/Nicole/Butterfly/Butterfly

```diff
@@ -37,7 +37,7 @@ and watched it with a smile
 
 Soon the quarreling was over
 There was peace throughout the land
-Making all the days of travelling
+Making all the days of traveling
 seem worth while
```

Comment:
I think "worth while" should be one word here as well

Reply:
Ok, @snshn I'll make that change. Btw, you might like to check out option #6 on this page. It's a nice timesaver for reviewers when reviewing ;) https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request

Once that's used, it appears like this (3) https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/incorporating-feedback-in-your-pull-request#applying-suggested-changes

Merry Christmas, btw :)

Reply:
Merry Xmas to you too, Andy!